pkg/scheduler/scheduler_test.go

```diff
@@ -775,6 +825,9 @@ func TestSchedulerWithVolumeBinding(t *testing.T) {
 	findErr := fmt.Errorf("find err")
```

Comment:
Because volume binding code is moved into a separate plugin, probably this test can be moved into the plugin too. I can do it in a follow-up PR if it will be better.